### PR TITLE
Add incremental improvements of classier.

### PR DESCRIPTION
**Pull request by andrazhribernik**
I added @matichorvat changes from #2 as well.

### CODE REVIEW

File: tutorial.ipynb [hunk 1]
```diff
@@ -559,20 +509,16 @@
...
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "pipeline.predict([\"Literally just 8 incredible dog videos.\"])"
+    "pipeline.predict([\"10 things you need to do...\"])"
```
Comment (owlas) [inline tutorial.ipynb:534]: What is the benefit of this? Maybe it wasn't intentional.

File: tutorial.ipynb [hunk 2]
```diff
@@ -172,12 +156,11 @@
...
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "df['full_content'] = (df.description + df.title)\n",
+    "# df['full_content'] = (df.description + df.title)\n",
+    "df['full_content'] = (df.title)\n",
```
Comment (owlas) [inline tutorial.ipynb]: This is quite confusing. We can just use `df.title` in the notebook and then switch to `df.full_content` later to show the incremental benefit.

File: tutorial.ipynb [hunk 3]
```diff
@@ -582,6 +528,106 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "### Evaluate performance\n",
```
Comment (owlas) [inline tutorial.ipynb]: This is model selection based on the testing data!

File: tutorial.ipynb [hunk 4]
```diff
@@ -582,6 +528,106 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.metrics import accuracy_score\n",
+    "from sklearn.metrics import confusion_matrix\n",
+    "from utils.plotting import plot_confusion_matrix\n",
+    "\n",
+    "def evaluate(pipeline, test_samples, test_labels, n=5):\n",
```
Comment (owlas) [inline tutorial.ipynb]: The notebook would be tidier if this was moved into a separate file.

File: tutorial.ipynb [hunk 5]
```diff
@@ -590,15 +636,14 @@
...
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "steps = (\n",
-    "    ('vectorizer', CountVectorizer()),\n",
+    "    ('vectorizer', TfidfVectorizer()),\n",
```
Comment (owlas) [inline tutorial.ipynb]: Nice incremental addition

**Review by owlas — CHANGES_REQUESTED** [review]
Additional steps are great. Thanks for spell check too

File: tutorial.ipynb (hunk 1 above)
Comment (andrazhribernik) [reply tutorial.ipynb:534]: The original sentence was labeled as 0 when I switched from title + description into title only :)

**Review by owlas — APPROVED** [review]

**Pull request merged**